plugins/modules/aci_epg_useg_attribute_block_statement.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Akini Ross <akinross@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_epg_useg_attribute_block_statement
+short_description: Manage EPG useg Attributes Block Statements
+description:
+- Manage EPG useg Attributes Block Statements (fv:SCrtrn)
```

Comment:
can you add the class to the short description instead of the description for consistency with the other modules?